vibegit/cli.py

```diff
@@ -748,9 +753,14 @@ def prompt_main_workflow(self):
         elif mode == "interactive":
             self.run_interactive_commit_workflow()
 
-    def run_commit_workflow(self):
+    def run_interactive_commit_workflow(self):
+        # ... rest of the method remains the same ...
+
+    def run_commit_workflow(self, debug: bool = False):
```

Comment:
```suggestion
    def run_commit_workflow(self, debug: bool = False):
```
fafuq? why did it make this change? thats not needed is it?